processYear.ts

```diff
@@ -192,13 +194,24 @@ export const processClustersForYear = async (
       const lca = await runLca(lcaInputs);
       // console.log(lca);
       results.lcaResults = lca;
-      // $ / dry metric ton
+      // $ / wet metric ton
       const fuelCost =
         (results.totalFeedstockCost + results.totalTransportationCost + results.totalMoveInCost) /
         results.totalFeedstock;
       // return updated fuel cost so that tea results can be updated later
       results.fuelCost = fuelCost;
 
+      const moistureContentPercentage = params.moistureContent / 100.0;
+
+      // calculate dry values ($ / dry metric ton)
+      results.totalDryFeedstock = results.totalFeedstock * (1 - moistureContentPercentage);
+      results.totalDryCoproduct = results.totalCoproduct * (1 - moistureContentPercentage);
+
+      results.harvestCostPerDryTon = results.totalFeedstockCost / results.totalDryFeedstock;
+      results.transportationCostPerDryTon = results.totalTransportationCost / results.totalDryFeedstock;
+      results.moveInCostPerDryTon = results.totalMoveInCost / results.totalDryFeedstock;
+      results.totalCostPerDryTon = results.fuelCost / results.totalDryFeedstock;
```

Comment:
`fuelCost` is in $/green ton, and we should divide the total cost by `totalDryFeedstock`.
`results.totalCostPerDryTon = (results.totalFeedstockCost + results.totalTransportationCost + results.totalMoveInCost) / results.totalDryFeedstock;`